eZ/Publish/Core/Search/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -176,8 +180,12 @@ protected function getResultCount(Criterion $filter, $fieldFilters)
     protected function getContentInfoList(Criterion $filter, $sort, $offset, $limit, $fieldFilters)
     {
         $query = $this->handler->createSelectQuery();
-        $query->selectDistinct(
+        $query->select(
```

Comment:
not a safe change unless we only deal with one language at a time (including attributes/fields and named data).